qiita_pet/support_files/doc/source/tutorials/account-creation.rst

```diff
@@ -1,64 +0,0 @@
-.. _account-creation:
```

Comment:
CMI workshop doc (http://cmi-workshop.readthedocs.io/en/latest/qiita-create-study.html#signing-up-for-a-qiita-account) is not as comprehensive as this one.
Do you think that we should get rid of retrieving password info, etc.?